tests/unittests/test_util.py

```diff
@@ -3226,10 +3226,18 @@ def test_happy_path(self, in_data, expected):
         assert expected == util.maybe_b64decode(in_data)
 
 
+class MockPath:
+    def __init__(self, target_file="/does/not/exist"):
+        self.target_file = target_file
+
+    def get_cpath(self, *args):
+        return self.target_file
```

Comment:
Probably overkill, but none of these tests assert we are specifically looking at hotplug.enabled in get_cpath. So, our implementation of read_hotplug_enabled_file  could change to `get_cpath("warnings")` with no test failures. 
Can we assert we are validating the specific hotplug.enabled lookup in Paths  somehow in these tests. Maybe just something like:
```diff
--- a/tests/unittests/test_util.py
+++ b/tests/unittests/test_util.py
@@ -3231,6 +3231,9 @@ class MockPath:
         self.target_file = target_file
 
     def get_cpath(self, *args):
+        assert args == (
+            "hotplug.enabled",
+        ), f"Invalid get_cpath argument {args}"
         return self.target_file

```